templates/server.conf.erb

```diff
@@ -55,6 +65,8 @@ ssl_certfile: <%= scope['pulp::db_ssl_certfile'] %>
 verify_ssl: <%= scope['pulp::db_verify_ssl'] %>
 ca_path: <%= scope['pulp::db_ca_path'] %>
 <% end -%>
+# unsafe_autoretry: false
+# write_concern: majority
```

Comment:
Agreed, the data_repeaing was overlooked, and the ldap, to be fair, is deprecated. So, I would vote for adding them when the parameters are added. As that avoids missing them in the future, and its fairly easy to do.